lib/index.js

```diff
@@ -15,6 +15,7 @@ const renderTemplate = require('./utils/renderTemplate')
 const emoji = require('./emoji')
 const inspect = require('./utils/inspect')
 const validateConfig = require('./validateConfig')
+const minifyHTML = require('./utils/minifyHTML')
```

Comment:
Please make it `minifyHtml` not `minifyHTML` (that's a typical convention for abbreviations).